contracts/earn/strategies/Aave/StrategyAave.sol

```diff
@@ -0,0 +1,139 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.6.12;
+
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
+import "@uniswap/lib/contracts/libraries/TransferHelper.sol";
+
+import "../../../interfaces/Aave/IAavePool.sol";
+import "../../../interfaces/Aave/IAaveV3Incentives.sol";
+import "../../../interfaces/Aave/IDataProvider.sol";
+
+import "../BaseStrategy.sol";
+
+contract StrategyAave is BaseStrategy {
+    using SafeERC20 for IERC20;
+    using SafeMath for uint256;
+
+    /* ----- State Variables ----- */
+
+    address[] public rewardAssets;
+    uint256 public lastHarvest;
+
+    /* ----- Constant Variables ----- */
+
+    address public constant aavePool = address(0x794a61358D6845594F94dc1DB02A252b5b4814aD); // Aave: Pool V3
+    address public constant incentivesController = address(0x929EC64c34a17401F460460D4B9390518E5B473e); // Aave: Incentives V3
+    address public constant dataProvider = address(0x69FA688f1Dc47d4B5d8029D5a35FB7a548310654); // Aave: Pool Data Provider V3
```

Comment:
@0xmer1in  could you please help double check these addresses in deployment?

Reply:
both correct